pkg/mobile/integration/service.go

```diff
@@ -33,22 +33,41 @@ var capabilities = map[string]map[string][]string{
 	"fh-sync-server": map[string][]string{
 		"capabilities": {"data storage, data syncronisation"},
 		"integrations": {"keycloak", "mcp-mobile-keys", "3scale"},
+		"description": {
+			"This service effectively allows FeedHenry Apps to seamlessly continue working when the network connection is lost, " +
+				"and allow them to recover when the network connection is restored.",
+			"- Allows mobile apps to use and update data offline",
+			"- Manages bi-directional data synchronization from multiple client apps",
+			"- Distributes data updates to connected clients",
+			"- Manages data collisions",
+		},
 	},
 	"keycloak": map[string][]string{
 		"capabilities": {"authentication, authorisation"},
 		"integrations": {"fh-sync"},
+		"description": {
+			"Keycloak is an open source Identity and Access Management solution aimed at modern applications and services. " +
+				"It makes it easy to secure applications and services with little to no code. ",
+		},
 	},
 	"mcp-mobile-keys": map[string][]string{
 		"capabilities": {"access apps"},
 		"integrations": {},
+		"description":  {"The api key generated for your mobile applications that can be used by a client and service to authenticate users."},
 	},
 	"3scale": map[string][]string{
 		"capabilities": {"authentication, authorization"},
 		"integrations": {},
+		"description": {
+			"3scale is the API infrastructure to build on now, and for the future. " +
+				"3scale makes it easy to manage your APIs for internal or external users. " +
```

Comment:
just wondering... is there no other way, than checking in these descriptions to the MCP ?

Reply:
you could potentially look up the service class from the catalog in the UI and use the description that is there

Reply:
The description in the serviceClass does not seem to be popluated from the APB.yml file. For example, in keycloak:
https://github.com/feedhenry/keycloak-apb/blob/master/apb.yml#L3

And in the serviceClass:
```
Object { id: "keycloak", name: "keycloak", ..., description: "", ...}
```

Reply:
seems to work for 3scale https://github.com/feedhenry/3scale-apb/blob/master/apb.yml#L3

```
 "spec": {
        "clusterServiceBrokerName": "ansible-service-broker",
        "externalName": "dh-3scale-apb",
        "externalID": "2b825339e8d685a78476621a252beea8",
        "description": "Unlock the power of your APIs. This will deploy Apicast to your Openshift cluster and connect  it to a dedicated Redis. For this to function correctly it needs to be configured with a token and domain for the account it should pull it's configuration values from (usually in 3scale.net).\nYou can create an account with 3scale.net here: https://www.3scale.net/signup/\n",
        "bindable": true,
```